app/src/main/java/org/fossasia/openevent/app/common/di/module/BarcodeModule.java

```diff
@@ -26,45 +20,4 @@
         return PublishSubject.create();
     }
 
-    @Provides
-    @Singleton
-    Detector.Processor<Barcode> providesProcessor(@Named("barcodeEmitter") PublishSubject<Notification<Barcode>> emitter) {
-        return new Detector.Processor<Barcode>() {
-            @Override
-            public void release() {
-                // No action to be taken
-            }
-
-            @Override
-            public void receiveDetections(Detector.Detections<Barcode> detections) {
-                SparseArray<Barcode> barcodeSparseArray = detections.getDetectedItems();
-                if (barcodeSparseArray.size() == 0)
-                    emitter.onNext(Notification.createOnError(new Throwable()));
-                else
-                    emitter.onNext(Notification.createOnNext(barcodeSparseArray.valueAt(0)));
-            }
-        };
-    }
-
-    @Provides
-    BarcodeDetector providesBarCodeDetector(Context context, Detector.Processor<Barcode> processor) {
-        BarcodeDetector barcodeDetector = new BarcodeDetector.Builder(context)
-            .setBarcodeFormats(Barcode.QR_CODE)
-            .build();
-
-        barcodeDetector.setProcessor(processor);
-
-        return barcodeDetector;
-    }
-
-    @Provides
-    CameraSource providesCameraSource(Context context, BarcodeDetector barcodeDetector) {
-        return new CameraSource
-            .Builder(context, barcodeDetector)
-            .setRequestedPreviewSize(640, 480)
-            .setRequestedFps(15.0f)
-            .setAutoFocusEnabled(true)
-            .build();
-    }
```

Comment:
Removing this is not necessary, inject it wherever needed, and if they are not needed, then replace them with needed dependencies